packages/block-editor/src/components/writing-flow/use-multi-selection.js

```diff
@@ -15,16 +15,19 @@ import { useSelect } from '@wordpress/data';
 import { store as blockEditorStore } from '../../store';
 import { __unstableUseBlockRef as useBlockRef } from '../block-list/use-block-props/use-block-refs';
 
-function toggleRichText( container, toggle ) {
-	Array.from( container.querySelectorAll( '.rich-text' ) ).forEach(
-		( node ) => {
-			if ( toggle ) {
-				node.setAttribute( 'contenteditable', true );
-			} else {
-				node.removeAttribute( 'contenteditable' );
-			}
+export function toggleRichText( container, toggle ) {
+	Array.from(
+		container.querySelectorAll(
+			// Exclude the Post Title from multi-select disable.
+			'.rich-text:not( .editor-post-title__input )'
```

Comment:
@ellatrix could we solve this issue without adding the selector from `editor` package to `block-editor`? Any thoughts?

Reply:
Yes. :) We should make the post title a block and lots of issues will be resolved. ;)